backends/graphics3d/openglsdl/openglsdl-graphics3d.cpp

```diff
@@ -262,7 +262,7 @@ void OpenGLSdlGraphics3dManager::createOrUpdateScreen() {
 		g_system->quit();
 	}
 
-#ifdef USE_GLEW
+#if defined(USE_GLEW) && !defined(NINTENDO_SWITCH)
```

Comment:
why glewinit is moved?

Reply:
Directly after "_glContext = SDL_GL_CreateContext(_window->getSDLWindow());", there is a call to glClear, which is failed until glewInit si called.
It is the glew library that supports links to OpenGL 1.1 functions on Switch port.

Reply:
can you put some comment on this, here and below?

Reply:
Done